src/codegen/code_context.cpp

```diff
@@ -158,18 +159,15 @@ void CodeContext::RegisterFunction(llvm::Function *func) {
   functions_.emplace_back(func, nullptr);
 }
 
-void CodeContext::RegisterExternalFunction(
-    llvm::Function *func_decl, UNUSED_ATTRIBUTE llvm::Function *external,
-    CodeContext::FuncPtr func_impl) {
+void CodeContext::RegisterExternalFunction(llvm::Function *func_decl,
+                                           CodeContext::FuncPtr func_impl) {
   PL_ASSERT(func_decl->isDeclaration() &&
             "The first argument must be a function declaration");
-  PL_ASSERT(!external->isDeclaration() &&
```

Comment:
Why remove these checks?

Reply:
I only require a `llvm::Function *func_decl` to get the _function_name_ and the `void *` pointing to the _function_.